R/pr.R

```diff
@@ -0,0 +1,556 @@
+#' Create a new Plumber router
+#'
+#' @param filters a list of plumber filters
+#' @param file path to file to plumb
+#' @param envir an environment to be used as the enclosure for the routers execution
+#'
+#' @return A new `plumber` router
+#'
+#' @examples
+#' \dontrun{
+#' pr() %>%
+#'   pr_run()
+#' }
+#'
+#' @export
+pr <- function(file = NULL,
+               filters = defaultPlumberFilters,
+               envir = new.env(parent = .GlobalEnv)) {
+  plumber$new(file = file, filters = filters, envir = envir)
+}
+
+#' Add handler to Plumber router
+#'
+#' This collection of functions creates handlers for a Plumber router.
+#'
+#' The generic [pr_handle()] creates a handle for the given methods. Specific
+#' functions are implemented for the following HTTP methods:
+#' * `GET`
+#' * `POST`
+#' * `PUT`
+#' * `DELETE`
+#' * `HEAD`
+#' Each function mutates the plumber router in place, but also invisibly returns
+#' the updated router.
+#'
+#' @param pr A plumber router
+#' @param methods Character vector of HTTP methods
+#' @param path The endpoint path
+#' @param handler a handler function
```

Comment:
Consistent use of capitalization, please

Reply:
Ahh good catch. Thank you!